### ISSUE TITLE
No activity?

### ISSUE THREAD

**Issue by paulcanning** [scribu/wp-posts-to-posts#535]
Looks like this plugin is dead in the water?

Anyone know if someone else has forked it to an active repo, or if there is a new post-2-post plugin?

**Comment by meloniq** [comment]
Why dead?! Maybe stable enough? :)

**Comment by paulcanning** [comment]
Well I posted issues over a week ago, without any answer.

There are pull requests that are long overdue.

Authors website hasn't been updated in a year, and this plugin is also about a year old without updates.

Yes, it may be stable, but its not complete...

**Comment by meloniq** [comment]
That shortly shows that there are more people willing to ask questions, rather than those who willing to answer them... means, a standard situation :)

**Comment by jdorner4** [comment]
On the plugin page at: https://wordpress.org/plugins/posts-to-posts/
it says "I, scribu, will not be offering support (either free or paid) for this plugin anymore.
If you want to help maintain the plugin, fork it on github and open pull requests."

**Comment by scribu** [comment]
Yeah, I found that even reviewing pull requests is a lot of work.

I updated the README: d2f297dbe257a09438da6ffe22c9a876acb7b49b

**Comment by danielbachhuber** [comment]
@scribu I can take things over, if you'd like.

**Comment by scribu** [comment]
Thanks, @danielbachhuber, but I've heard that line several times before, from several different people, and nothing happened.

If someone is truly interested, they'll just fork it, merge any interesting pull requests and go from there.

**Comment by danielbachhuber** [comment]
> If someone is truly interested, they'll just fork it, merge any interesting pull requests and go from there.

Sounds good. I might actually start with a rename, actually.

**Comment by scribu** [comment]
Yeah, it has sort of outgrown the name. Go nuts. :)

**Comment by jazbek** [comment]
@danielbachhuber can you keep us posted if this happens? would be really excited to see this plugin become active again, and don't want to miss your fork...

**Comment by danielbachhuber** [comment]
> can you keep us posted if this happens?

Yep, will do.

**Comment by JiveDig** [comment]
+1 on a takeover! This plugin is extremely useful/powerful and I'm thankful for the years of work Scribu has into it. There is currently no equal alternative so I'd hate to see it die. 

I'm a happy (but scared) user of the plugin, a mediocre (backend) dev, so I don't have the skill set required to takeover. 

3 cheers for Daniel forking and resurrecting!

**Comment by aaronjorbin** [comment]
@danielbachhuber If you want to start it as an organization, happy to lend a hand as well.

**Comment by pippinsplugins** [comment]
@danielbachhuber @aaronjorbin I'd be happy to contribute as well. Can't say I'll have much time for it but it's important to me that this stays alive.

**Comment by staylor** [comment]
Add me to the org if you create one @danielbachhuber et al

**Comment by tnorthcutt** [comment]
@danielbachhuber +1 happy to help out

**Comment by swissspidy** [comment]
@danielbachhuber you can count me in as well

**Comment by mpeshev** [comment]
+1

**Comment by efc** [comment]
@danielbachhuber +1, happy to pay, happy to help, let us know what you need, we use this extensively on a magazine site.

**Comment by brichards** [comment]
posts-to-posts and CMB2 are my go-to libraries when building nearly any site.

@scribu Thanks for putting in so much effort across such a large body of work over so many years. I know you're on to different things that are hopefully very meaningful to you, and I continue to wish you the best with that. Nevertheless, your contributions are sorely missed and I continue to look forward to your celebrated return :)

**Comment by ghost** [comment]
Has anyone used https://wordpress.org/plugins/post-connector/? A fork, perhaps?

**Comment by swissspidy** [comment]
No experience with Post Connector, but it also hasn't seen many updates. Not a fork of Posts 2 Posts AFAIK.

**Comment by JiveDig** [comment]
Barry is a great dev, but the plugin is much different. It uses CPT's with post_meta for the connections. It also doesn't allow user connections. It's just too different to be considered a replacement. For smaller sites it's probably cool, similar to ACF (and similar plugins) relationship or post object field types.

**Comment by thinkstylestudio** [comment]
Is this guy up for adoption? I'm sure this question has been raised previously.

**Comment by ghost** [comment]
> Is this guy up for adoption? I'm sure this question has been raised previously.

https://github.com/scribu/wp-posts-to-posts/issues/535#issuecomment-234041508

**Comment by toshotosho** [comment]
Automattic should take over and add it to core.

@danielbachhuber maybe another kickstarter campaign?

**Comment by ejntaylor** [comment]
How did the fork come along?

**Comment by wgroenewold** [comment]
@danielbachhuber Would love to get this fork started. I'm thinking about adding multisite support (being able to connect posts between sites, not just inside one site) and adding better rights management for this plugin. Would love your thoughts about that.

**Comment by yaronguez** [comment]
Whatever happened here? @danielbachhuber did you end up taking over? Is this plugin still stable to use with the latest version of WP?

**Comment by swissspidy** [comment]
The plugin is still stable to use with the latest version of WordPress.

As suggested in the readme:

> If you want to fix a bug in the plugin or add new features, feel free to fork it on github. I will not be accepting any pull requests though, since ensuring that a code change doesn't break things takes effort, effort which I'm no longer willing to expend.

There's https://github.com/wp-posts-to-posts/wp-posts-to-posts with a bunch of people in that organization already.

We should probably just start submitting pull requests there for whatever bugs we encounter and features we like

**Comment by corradomatt** [comment]
I take it this is officially dead now?

@yaronguez - maybe we should get someone(s) from AWP to adopt this project?

**Comment by ejntaylor** [comment]
I wouldn't call it dead. Just without a core team/leadership.

@swisspidy referenced this fork: https://github.com/wp-posts-to-posts/wp-posts-to-posts

How about:

- We make that fork the main place for ongoing development of the plugin
- I've asked Scott Taylor (who I think is current admin of Fork) to let me become an admin -- I'll then enable issues on the project so we can continue the conversation there.
- We should then ask Scribu to link to that fork and also change readme to encourage others to get involved.

That should help kick start some life into this (imho) really important plugin.

Does that sound like a good course of action to everyone listening in?

@corradomatt - whos AWP?

**Comment by corradomatt** [comment]
Thanks @raisonon - sounds like a good plan to me.

AWP (Advanced WordPress) is a meetup group started in SD a few years ago that @yaronguez and I are organizers for - https://www.meetup.com/Advanced-WordPress/ -- there's also a Facebook group that is huge now - https://www.facebook.com/groups/advancedwp/

**Comment by JiveDig** [comment]
It's worth noting that JJJ just started https://github.com/stuttter/wp-relationships. If there is interest here for contributors maybe it's worth working on a newer, more robust solution?

**Comment by ejntaylor** [comment]
Quite keen on wp-relationships, except already have projects built with p2p. 

He does write that he thought starting again because of limitations with p2p, so am kind of keen to get involved. Not really sure if there is any protocol to introduce ourselves as contributors other than just cracking on with PRs?

**Comment by JiveDig** [comment]
He's very open to talking about it. See https://twitter.com/JJJ/status/923203747836301312
If there is enough interest (and there should be IMO), we should consider a migration tool.

**Comment by JJJ** [comment]
I don't want to poach anyone away from P2P (especially since WPR isn't done-done by any means.)

I'm definitely open to letting a group of people run with WPR.

**Comment by rilwis** [comment]
Hi everyone, we have developed an alternative solutions for P2P. Check it out here:

https://metabox.io/plugins/mb-relationships/

The plugin is inspired by P2P, the API and database is similar. We also use Meta Box to handle the meta boxes in the back-end. So, the code base is very lightweight.

Any feedback is welcomed.

**Comment by ejntaylor** [comment]
This looks great @rilwis but I think this functionality is quite central to improving WP as an enterprise solution so would prefer to use/contribute to an FOSS solution like @JJJ is spearheading.

**Comment by JiveDig** [comment]
Agreed. Although @rilwis's solution looks good for his customers and probably other solutions, the real problem is the lack of scalable object to object relationships in WP. A solid, standalone solution is much needed. I wouldn't even mind paying for it if it was stable and well maintained.

**Comment by PayteR** [comment]
Hi everybody, any news regarding this? For last few days i have made my own resarch of posts relation solutions for WP, and P2P plugin is the best, because it's not depended on other plugins and have great API, it's a shame, that it's dead... every fork that i saw was dead too, let me know, if im wrong, please.

**Comment by paulcanning** [comment]
I've forked it with the intention to eventually work on it. I'd need to collate a list of feature requests to see what people want.
